sceptre/securitycentral/config/prod/sechub_suppress_infra.yaml

```diff
@@ -0,0 +1,3 @@
+template:
+  path: sechub_suppress_infra.yaml
```

Comment:
I think it would make more sense to deploy using orgformation at https://github.com/Sage-Bionetworks-IT/organizations-infra/tree/master/org-formation/075-security-hub

Also would be nice to add some information to the README file to document how this works and where you got it from.

Reply:
Done